src/background.js

```diff
@@ -41,16 +41,12 @@ const KEYWORDS = {
     "2019",
```

Comment:
we are very close to 2020 and the experiment will run in 2020
we should maybe also add "www.getfirefox.com if we have "www.firefox.com"